packages/netlify-cms-ui-default/src/Dropdown.js

```diff
@@ -128,4 +129,26 @@ DropdownItem.propTypes = {
   className: PropTypes.string,
 };
 
-export { Dropdown as default, DropdownItem, DropdownButton, StyledDropdownButton };
+const DropdownCheckedItem = ({ label, id, checked, onClick }) => {
+  return (
+    <StyledMenuItem isCheckedItem={true} isActive={checked} onClick={onClick}>
+      <input readOnly checked={checked} type="checkbox" id={id} />
```

Comment:
Could we wrap this in a styled element like `StyledCheckbox` and add a `margin-right: 10px` to it just for the visual appeal?

Reply:
@hanneskuettner You mean to say wrap `input` tag or `DropdownCheckedItem` in `DropdownCheckbox`?

Reply:
The input tag